### ISSUE TITLE
igor: QinQ segmentation

### ISSUE THREAD

**Issue by floren** [sandia-minimega/minimega#878]
Implement network segmentation between different reservations using QinQ.

**Comment by floren** [comment]
I think we need to be able to bridge the head node and/or an NFS server into the QinQ segment

**Comment by jcrussell** [comment]
I've run into this issue on my slice of nodes -- the head node can't give the 10G interface an IP because it is not on the same VLAN.

**Comment by jcrussell** [comment]
Would this work? For each new reservation, we assign a VLAN. The nodes in the VLAN will have their ports set to `access vlan X`. On the headnode, we add a new vlan-device (`vconfig add ethN X`), set the IP (I think we should be able to reuse the same IP) and up the interface. `dnsmasq` should then serve DHCP without further changes:

```
On  systems  which support it, dnsmasq binds the wildcard address, even when it
is listening on only some interfaces. It then discards requests that it
shouldn't reply to. This has the advantage of working even when interfaces come
and go and change address.
```

**Comment by djfritz** [comment]
Yep. There was an offline discussion that arrived at exactly this. I'll
give it a try.

**Comment by floren** [comment]
Implemented by #904
